CONTRIBUTING.md

```diff
@@ -0,0 +1,609 @@
+# AWS iOS SDK Contributing Guide
+
+Thank you for your interest in contributing to our project! <3 Whether it's a bug report, new feature, correction, or additional documentation, we greatly value feedback and contributions from our community. The following is our contribution guide, which we hope you will read through carefully prior to submitting a pull-request (PR) or issue. In the event that our guide is not up to date, feel free to let us know by opening an issue (or even better, submit a PR with your proposed corrections ;)).
+
+- [Getting Started](#getting-started)
+- [Tools](#tools)
+  - [Integration Testing Set Up](#integration-testing-setup)
+- [Integrated Frameworks](#integrated-frameworks)
+- [Workflows](#workflows)
+  - [New Features or Bug Fixes](#New-Features-or-Bug-Fixes)
+- [Debugging](#debugging)
+  - [Running Cocoapods Locally](#running-cocoapods-locally)
+- [Pull Requests](#pull-requests)
+  - [Pull Request Checklist](#pull-request-checklist)
+  - [Step 1: Open Issue](#step-1-open-issue)
+  - [Step 2: Design (optional)](#step-2-design-optional)
+  - [Step 3: Fork The Repo](#step-3-fork-the-repo)
+  - [Step 4: Work your Magic](#step-3-work-your-magic)
+  - [Step 5: Commit](#step-4-commit)
+  - [Step 6: Pull Request](#step-5-pull-request)
+  - [Step 7: Merge](#step-6-merge)
+- [Troubleshooting](#troubleshooting)
+- [Related Repositories](#related-repositories)
+- [Finding Contributions](#finding-contributions-to-work-on)
+- [Code of Conduct](#code-of-conduct)
+- [Security Issue Notifications](#security-issue-notifications)
+- [Licensing](#licensing)
+
+## Getting Started
+
+To get started with contributing to the AWS iOS SDK, first make sure you have the latest version of [Xcode](https://developer.apple.com/xcode/) installed. There are many different ways to build frameworks for iOS developers but we love to use [Cocoapods](https://guides.cocoapods.org/using/getting-started.html) and these instructions will include how to get started with Cocoapods so make sure that is installed as well if you'd like to follow along. You can install cocoapods by simply running:
```

Comment:
Wording: maybe, `To start contributing to the AWS iOS SDK`?

Reply:
fixed